### DISCUSSION TITLE
Welcome to java-gtk Discussions!

### DISCUSSION THREAD

**Discussion by bailuk** [bailuk/java-gtk#12] [Announcements]
Use this forum to ask questions and share knowledge about java-gtk

**Comment by dadkins789** [comment]
Was working with Cairo ImageSurface context and was wondering why cairo_select_font_face and cairo_set_font_size don't appear to be present. What is the recommended method of setting up a font with a context for an ImageSurface.

**Comment by bailuk** [comment]
These functions where missing because no one needed them before. `cario-custom.gir` is the only manually created gir-File in java-gtk.
Font rendering in GTK is usually done trough lib pango. There is an example: `PangoTextMaks.java`.
